examples/anim.rs

```diff
@@ -134,7 +134,7 @@ fn main() {
     let blend_time = 0.2f32;
     let mut timer = win.input.time();
 
-    while win.update() && !three::KEY_ESCAPE.is_hit(&win.input) {
+    while win.update() && !win.input.button(three::KEY_ESCAPE).pressed {
```

Comment:
hmm, that worries me. The new code is more verbose, while we need to make it shorter

Reply:
From my point of view, code is little more verbose but much clearer now. If we're talking about amount of symbols, I don't see any way to reduce it atm.